validator/main.go

```diff
@@ -79,6 +80,7 @@ func main() {
 func validate(vConfig models.ValidateConfig) error {
 	var err error
 	for i := 0; i < awsservice.StandardRetries; i++ {
+		basic.RetryCount = i + 1
```

Comment:
where is this var being used? and for what?

Reply:
Oh nice catch! That will be removed I was using it for testing purposes